frontend/src/components/Input/index.js

```diff
@@ -0,0 +1,43 @@
+import { PropTypes } from "prop-types";
+import styles from "./index.module.css";
+
+const Input = ({ inputType, disabled, error }) => {
+  let texts = {
+    text: { placeholder: "Ariana", errorMessage: "Enter your name." },
+    email: {
+      placeholder: "arina@fortunemail.com",
+      errorMessage: "Invalid email.",
+    },
+  };
+
+  return (
+    <>
+      {error ? (
+        <div className={styles.container}>
+          <span>{texts[inputType].errorMessage}</span>
```

Comment:
Can't we conditionally render the `error` instead of duplicating the `input`?

Reply:
Yes, I changed to:
```
<div className={!error ? "" : styles.errorContainer}>
      {!error ? "" : <span>{errorMessage}</span>}
      <input
        disabled={disabled}
        type={type}
        placeholder={placeholder}
        className={
          !error ? styles.input : `${styles.input} ${styles.inputError}`
        }
      />
</div>
```